app/templates/content.hbs

```diff
@@ -1,9 +1,12 @@
-{{title model.title}}
+{{title node.title}}
 
 <div id="view-page">
     <div class="p-t-xl m-t-xl p-b-md m-b-md dark-overlay-header-background header-row">{{!HEADER ROW}}
         <div class="container">{{!CONTAINER DIV}}
-            <h1 id="preprintTitle" class="p-b-md">{{model.title}}</h1>
+            <h1 id="preprintTitle" class="p-b-md">{{node.title}}</h1>
+            {{#if isAdmin}}
+                <a type="button" class="pull-right btn btn-success m-h-lg" href={{concat model.links.html '?edit'}}  {{action "click" 'link' "Content - Edit Preprint" (concat model.links.html '?edit')}}>{{t "content.project_button.edit_preprint"}}</a>
```

Comment:
Should use link-to with query params instead of a tag (see [Ember docs](https://guides.emberjs.com/v2.7.0/routing/query-params/#toc_link-to-helper) example)